uint64 timestamp
string name
Position pos
uint32 price